src/temp/RunConfigurationPanel/components/RunConfigurationPanelTests/core/runConfigurationPanelTests.constants.tsx

```diff
@@ -0,0 +1,18 @@
+import { ISelectInputProps } from '@/components/Inputs/SelectInput'
+
+export const FIELD_OPTIONS: ISelectInputProps['options'] = [
+  { value: 'ResponseTimes.Mean', display: 'Mean response time' },
+  { value: 'ResponseTimes.Max', display: 'Maximum response time' },
+  { value: 'ResponseTimes.Min', display: 'Minimum response time' },
+  { value: 'RequestFailureCount', display: 'Number of requests that failed' },
+  { value: 'RequestSuccessCount', display: 'Number of successfull requests' },
+]
```

Comment:
I do not think this is maintainable as we already have an enormous amount of combinations available (every path reachable on the metrics object), for example : 

```js
"ResponseTimes.Mean" // nested path
"RequestSuccessCount" // methods
"StatusCodesDistribution.404" // map with number as keys
"RequestEventTimes.FirstResponseByte.Mean" // map with string as keys
"RequestFailures.1.Reason" // nested path on an array
```

 And this declined for **each** path and for **each** map's key and **each** array's element.

Plus it is actually case insensitive
```js
"responsetimes.MEAN" == "ResponseTimes.Mean"
```

I strongly suggest we go for an `TextInput` over a `SelectInput` and simply do a regex validation (or not at all as a first iteration, the error would be caught by the engine).

Reply:
On second thoughts, what do you think of restraining the choice to 4-5 pre-selected metrics on the beginning. It seems more UX friendly to me since it will prevent the user from trying to launch a run and receive an error that could have been avoided with the right UI. However, this solution is temporary, just until we succeed to have a complete working user "journey". Then I'll develop a dedicated UI component which will be a text input with autocomplete but which only allows to pick a value from the suggestion and won't let the user go further if he only just entered text.